lib/oli/publishing.ex

```diff
@@ -503,6 +522,19 @@ defmodule Oli.Publishing do
     )
   end
 
+  def project_working_publication_by_setion_list(project_slug, list_section_ids) do
```

Comment:
We have a typo in `setion`. I think this function may be private since it is only called from another function in this same module. And what do you think about testing `query_unpublished_revisions_by_type_and_section/3`?